assets/js/components/ErrorNotice.js

```diff
@@ -98,26 +98,36 @@ export default function ErrorNotice( {
 	const reconnectURL = error?.data?.reconnectURL;
 
 	if ( reconnectURL && isURL( reconnectURL ) ) {
-		errorMessageWithModifications = createInterpolateElement(
+		errorMessageWithModifications =
+			errorMessageWithModifications +
+			' ' +
 			sprintf(
-				/* translators: %1$s: Original error message */
+				/* translators: %s: Reconnect URL */
 				__(
-					'%1$s To fix this, <a>redo the plugin setup</a>.',
+					'To fix this, <a href="%s">redo the plugin setup</a>.',
 					'google-site-kit'
 				),
```

Comment:
Why did we revert this translation improvement? We should be able to keep the `errorMessageWithModifications` inside the translation string at least, which is better from a localisation perspective.